lib/plugins/features/movies/controller.js

```diff
@@ -1,9 +1,32 @@
 'use strict';
 
 const Movie = require('../../../models/movie');
+const Movies = Movie.collection();
 
 exports.create = async (payload) => {
   const movie = await new Movie().save(payload);
 
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.find = async (query) => {
+  let movies;
+  if (query.title) {
+    movies = Movies.query((queryBuilder) => {
+      queryBuilder.where('title', '=', query.title);
+    }).fetch();
+  } else if (query.date1) {
```

Comment:
What do you think about having a separate query parameter for an exact match, and separate query parameters for a range? Also more descriptive names for the query parameters, e.g. 

`release_year` -> exact year match
`start_year` and `end_year` -> matching within `start_year <= X <= end_year` range